internal/commands/pull/command.go

```diff
@@ -60,32 +61,58 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
+	clientZipPkgs, err := cmd.inputs.resolveClientTemplates(ui, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	projectPath, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	proceed, err := checkAppDestination(ui, pathTarget)
+	var pathFrontend string
+	pathBackend := projectPath
+	if len(clientZipPkgs) != 0 {
+		pathFrontend = filepath.Join(projectPath, local.FrontendPath)
+		if proceed, err := checkPathDestination(ui, pathFrontend); err != nil {
```

Comment:
i don't think we need this check here, instead let's check the `pathProject` down below on L#87`